examples/native/scripts/generateEmbeddedDatabase.js

```diff
@@ -0,0 +1,27 @@
+/* this script needs @babel/node to run, check package.json script session */
+import * as fs from 'fs';
+import databasePromise from '../src/database/embeddedInit';
+
+import { generate100 } from '../src/model/generate';
```

Comment:
@sidferreira  where is the `generate100` function? It's not in here `https://github.com/Nozbe/WatermelonDB/tree/master/src/Model`, yet you import it from `import { generate100 } from '../src/model/generate';`

Reply:
@sidferreira Where is it?

Reply:
@Stophface ping me in twitter

Reply:
but if you check Root.js you will see that the generate code was already there in Model